.github/workflows/cpaas-sync-drivers.yml

```diff
@@ -73,14 +100,89 @@ jobs:
               gsutil -m cp -n -I "gs://${{ inputs.drivers-bucket }}/${{ matrix.platform }}/$(basename "${driver_version_dir}")/"
           done
 
+      # x86 support packages will not be uploaded to GCP.
+      # This is to prevent mixing it up with the upstream built packages.
+      - name: Generate 'latest' files
```

Comment:
```suggestion
      - name: Generate 'latest' and sha files
```

Reply:
Went with `Generate 'latest' and checksum files`, hope that's fine 🙂

Reply:
That's fine.